### PR TITLE
KAFKA-15448: Update upgrade guide for KIP-988

### PR DESCRIPTION
**Pull request by eduwercamacaro**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: docs/streams/upgrade-guide.html [hunk 1]
```diff
@@ -134,6 +134,12 @@ <h3 class="anchor-heading"><a id="streams_notable_changes" class="anchor-link"><
     </p>
 
     <h3><a id="streams_api_changes_370" href="#streams_api_changes_370">Streams API changes in 3.7.0</a></h3>
+    <p>
+        We added a new method to <code>KafkaStreams</code>, namely <code>KafkaStreams#setStandbyUpdateListener()</code> in
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-988%3A+Streams+Standby+Update+Listener">KIP-988</a>.
+        In which users can provide their customized implementation of the org.apache.kafka.streams.processor.StandbyUpdateListener interface to continuously monitor changes to standby tasks.
```
Comment (coltmcnealy-lh) [inline docs/streams/upgrade-guide.html]: > In which users can provide their customized implementation...

This is a subordinate clause, not an independent clause. I would propose replacing the `.` after `</a>` with a `,` and replacing `In` with `in`.

Comment (ableegoldman) [reply docs/streams/upgrade-guide.html]: +1 to Colt's suggestion, otherwise LGTM

File: docs/streams/upgrade-guide.html [hunk 2]
```diff
@@ -134,6 +134,12 @@ <h3 class="anchor-heading"><a id="streams_notable_changes" class="anchor-link"><
     </p>
 
     <h3><a id="streams_api_changes_370" href="#streams_api_changes_370">Streams API changes in 3.7.0</a></h3>
+    <p>
+        We added a new method to <code>KafkaStreams</code>, namely <code>KafkaStreams#setStandbyUpdateListener()</code> in
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-988%3A+Streams+Standby+Update+Listener">KIP-988</a>.
```
Comment (mjsax) [inline docs/streams/upgrade-guide.html]:
```suggestion
        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-988%3A+Streams+Standby+Update+Listener">KIP-988</a>,
```

File: docs/streams/upgrade-guide.html (hunk 1 above)
Comment (mjsax) [inline docs/streams/upgrade-guide.html]:
```suggestion
        in which users can provide their customized implementation of the newly added <code>StandbyUpdateListener</code> interface to continuously monitor changes to standby tasks.
```

Comment (eduwercamacaro) [reply docs/streams/upgrade-guide.html]: Thanks for the suggestions.

**Comment by mjsax** [comment]
Thanks for the PR!

Merged to `trunk` and cherry-picked to `3.7` branch.

**Pull request merged**